pkg/quarterdeck/db/models/apikeys_test.go

```diff
@@ -179,7 +179,7 @@ func (m *modelTestSuite) TestDeleteAPIKey() {
 	var permissions string
 	err = tx.QueryRow("SELECT permissions FROM revoked_api_keys WHERE id=$1 AND organization_id=$2", keyID, orgID).Scan(&permissions)
 	require.NoError(err, "could not fetched revoked key")
-	require.Equal(`["topics:create","topics:read","metrics:read","publisher","subscriber"]`, permissions, "permissions not serialized correctly")
+	require.Equal(`["projects:delete","apikeys:delete","apikeys:read","topics:create","topics:edit"]`, permissions, "permissions not serialized correctly")
```

Comment:
Ok, so these permissions changed because the permissions in the fixtures have new IDs - so we'll probably have to update the test fixtures with the right IDs. I think there are two - one in `pkg/quarterdeck/db/models/testdata` and one in `pkg/quarterdeck/testdata`.

Reply:
I sorted the fixtures out so this should be resolved now.